ocsp.go

```diff
@@ -428,15 +428,65 @@ func retryOCSP(
 			err:  fmt.Errorf("HTTP code is not OK. %v: %v", res.StatusCode, res.Status),
 		}
 	}
-	logger.Debug("reading contents")
 	ocspResBytes, err = io.ReadAll(res.Body)
 	if err != nil {
 		return ocspRes, ocspResBytes, &ocspStatus{
 			code: ocspFailedExtractResponse,
 			err:  err,
 		}
 	}
-	logger.Debug("parsing OCSP response")
+	ocspRes, err = ocsp.ParseResponse(ocspResBytes, issuer)
```

Comment:
are we able to test added code? e.g. with hang webserver?

Reply:
IMHO it is not worth it to write an automatic test, since this is such a specific case and it would require a lot effort. We already have a confluence page that describes how to manually test Privatelink related issues.